db/people.yml

```diff
@@ -2373,3 +2373,50 @@ htiwari:
       status: ms
   name: Hrishi Tiwari
   position: Former Masters Researcher
+kknox:
+  active: false
+  aka:
+    - Kevin Knox
+    - Knox, K.
+    - Knox, Kevin
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 2006
+      degree: PhD
+      institution: Columbia University
+      end_year: 2011
+    - begin_year: 2004
+      degree: MA in Physics
+      institution: Columbia University
+      end_year: 2006
+    - begin_year: 1997
+      degree: BA in Physics
+      institution: Cornell University
+      end_year: 2001
+  employment:
+    - begin_year: 2015
+      begin_month: 7
+      organization: J.P. Morgan
+      department: []
+      position: Vice President
+    - begin_year: 2011
+      begin_month: 7
+      end_month: 6
+      end_year: 2015
+      group: bg
+      status: postdoc
+      advisor: sbillinge
+      organization: Brookhaven National Laboratory
+      position: Post Doctoral Associate
+    - begin_year: 2006
+      begin_month: []
+      end_month: []
+      end_year: 2011
+      group: bg
+      status: Graduate Researcher
```

Comment:
this is from a list of known values.  I think you can figure out allowed values by looking at other entries in the db but let me know of you are having trouble